lib/cisco_node_utils/cisco_cmn_utils.rb

```diff
@@ -58,6 +58,21 @@ def self.symbol_to_cli(symbol)
     end
   end
 
+  # General utility class
```

Comment:
Hmm... merge said everything was up to date, so maybe I did it wrong.  I'll try to fix.